README.md

```diff
@@ -9,30 +9,46 @@
 ![workflow check](https://github.com/alibaba/hybridnet/actions/workflows/check.yml/badge.svg)
 ![workflow build](https://github.com/alibaba/hybridnet/actions/workflows/build.yml/badge.svg)
 
-## What is Hybridnet?
-
 Hybridnet is an open source container networking solution designed for hybrid clouds, integrated with Kubernetes and used officially by following well-known PaaS platforms,
 
 - ACK Distro of Alibaba Cloud
 - AECP of Alibaba Cloud
 - SOFAStack of Ant Financial Co.
 
-Hybridnet focuses on productive large-scale clusters, heterogeneous infrastructure and user-friendliness. Now hundreds of clusters are running hybridnet all over world.
+
+## Introduction
+
+Hybridnet is a Kubernetes CNI plugin implemented with "Policy Routing". The feature of "Policy Routing" is introduced in 2.1 version of linux kernel as a basic part of routing subsystem, which provides strong stability and compatibility.  
+
+Most CNI plugins classify the forms of container network into two types and make them work at their own paces without connection:
+1. *Overlay*, an abstract data plane on top of host network which is usually not visible to underlying network and brings cost
+2. *Underlay*, putting container traffic "directly" into host network and depending on the abilities of underlying network
+
+In hybridnet, we try to break the strict boundary of all the forms of container network with a simple design:
+1. Overlay and underlay network can be created in the same cluster.
+2. Once been created, underlay addresses are "registered" to overlay network with the same value. So every underlay address will use its "Overlay" identity to communicate with overlay addresses 
```

Comment:
Seems a bit elusive, IMO, underlay is using "Overlay" tunnel to communicate with overlay addresses, not identity

Reply:
fixed